website/docs/reference/commands/parse.md

```diff
@@ -3,32 +3,22 @@ title: "parse"
 id: "parse"
 ---
 
-<Changelog>
+## Overview
 
-New in `v0.19.0`
+The `dbt parse` command parses and validates the contents of your dbt project. If your project contains Jinja or YAML syntax errors, the command will fail.
 
-</Changelog>
+It will also produce an artifact with detailed timing information, which is useful to understand parsing times for large projects. For more details, [read about parsing](parsing).
```

Comment:
```suggestion
It will also produce an artifact with detailed timing information, which is useful to understand parsing times for large projects. Refer to [Project parsing](/reference/parsing) for more info.
```